tests/console/wavpack.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test basic functionality of wavpack audio compression format.
+# Maintainer: Ednilson Miura <emiura@suse.com>
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+use utils;
+
+sub run {
+    # setup
+    select_console 'root-console';
+    zypper_call 'in alsa alsa-utils wavpack';
+    assert_script_run("cp /usr/share/sounds/alsa/Noise.wav .");
```

Comment:
I think all of this could have been ran using a shell script (This avoids the need to run into typing issues if the worker host is under heavy load. Since you are not using anything else than assert_script_run, I guess it's feasible, wdyt?.